src/ReefGuideAPI.jl

```diff
@@ -55,26 +55,46 @@ Load regional data to act as an in-memory cache.
 OrderedDict of `RegionalCriteria` for each region.
 """
 function setup_regional_data(config::Dict)
+    reef_data_path = config["prepped_data"]["PREPPED_DATA_DIR"]
+
     if @isdefined(REGIONAL_DATA)
         @debug "Using previously generated regional data store."
         sleep(1)  # Pause so message is noticeably visible
+
+        reef_outline_path = joinpath(reef_data_path, "rrap_canonical_outlines.gpkg")
+        REGIONAL_DATA["reef_outlines"] = GDF.read(reef_outline_path)
+
+        REGIONAL_DATA["region_long_names"] = Dict(
+            "FarNorthern" => "Far Northern Management Area",
+            "Cairns-Cooktown" => "Cairns/Cooktown Management Area",
+            "Townsville-Whitsunday" => "Townsville/Whitsunday Management Area",
+            "Mackay-Capricorn" => "Mackay/Capricorn Management Area"
+        )
         return REGIONAL_DATA
     end
 
-    # Check disk-based store
+    # # Check disk-based store
     reg_cache_dir = config["server_config"]["REGIONAL_CACHE_DIR"]
     reg_cache_fn = joinpath(reg_cache_dir, "regional_cache.dat")
     if isfile(reg_cache_fn)
         @debug "Loading regional data cache from disk"
         @eval const REGIONAL_DATA = deserialize($(reg_cache_fn))
+
+        reef_outline_path = joinpath(reef_data_path, "rrap_canonical_outlines.gpkg")
+        REGIONAL_DATA["reef_outlines"] = GDF.read(reef_outline_path)
+
+        REGIONAL_DATA["region_long_names"] = Dict(
+            "FarNorthern" => "Far Northern Management Area",
+            "Cairns-Cooktown" => "Cairns/Cooktown Management Area",
+            "Townsville-Whitsunday" => "Townsville/Whitsunday Management Area",
+            "Mackay-Capricorn" => "Mackay/Capricorn Management Area"
+        )
```

Comment:
Isn't this info already in the canonical reefs data? Just wondering why we need it stored...

Reply:
Yes the long names are in the data already, but not the corresponding short names. I could change it to find which long name matches the first 3 characters of the target short name instead

Reply:
> matches the first 3 characters
I wouldn't do that, it feels hacky. `Cairns-Cooktown` is effectively our region id, so best to key on that as you have. 

@ConnectedSystems it's needed in `identify_potential_sites_edges`
```julia
    reef_lines = reef_lines[gdf.management_area .== region]
    gdf = gdf[gdf.management_area .== region, :]
```

As far as function design, having two different region string params is confusing.
```julia
function identify_potential_sites_edges(
    df::DataFrame,
    search_pixels::DataFrame,
    res::Float64,
    gdf::DataFrame,
    x_dist::Union{Int64,Float64},
    y_dist::Union{Int64,Float64},
    target_crs::GeoFormatTypes.CoordinateReferenceSystemFormat,
    reef_lines::Vector{Vector{GeometryBasics.Line{2,Float64}}},
    region::String;  # <-- how do I know what region id this wants?
    degree_step::Float64=15.0,
    n_rot_p_side::Int64=2,
    surr_threshold::Float64=0.33
)::DataFrame
```

 The fact that we have two different region strings is an implementation detail that should be hidden. I'll make separate comments.

Reply:
Yeah, the only reason the long name is in here is because in canonical_reefs we use the official GBRMPA spatial file to assign a region to each reef and they use the long names. It could be adjusted in there, or inside this identify_potential_sites_edges() function